tensorflow/lite/delegates/nnapi/java/src/main/java/org/tensorflow/lite/nnapi/BUILD

```diff
@@ -0,0 +1,7 @@
+licenses(["notice"])  # Apache 2.0
+
+filegroup(
+    name = "nnapi_delegate",
```

Comment:
maybe we should call this:

nnapi_delegate_srcs

Reply:
changed